src/pulse3D/plate_recording.py

```diff
@@ -160,18 +162,11 @@ def __init__(
             self.is_force_data = (
                 "y" in str(_get_excel_metadata_value(self._excel_sheet, TWITCHES_POINT_UP_UUID)).lower()
             )
-
-            self.noise_filter_uuid = (
-                TSP_TO_DEFAULT_FILTER_UUID[self.tissue_sampling_period] if self.is_magnetic_data else None
-            )
-            self.filter_coefficients = (
-                create_filter(self.noise_filter_uuid, self.tissue_sampling_period)
-                if self.noise_filter_uuid
-                else None
-            )
-
-            self.stiffness_factor = stiffness_factor if stiffness_factor else CARDIAC_STIFFNESS_FACTOR
-
+            for uuid_ in (PLATEMAP_NAME_UUID, PLATEMAP_LABEL_UUID):
```

Comment:
This is duplicate code from a few lines up, but it is ultimately not needed because these values will never be present in xlsx files. We should just set them to `NOT_APPLICABLE_LABEL`, there's no need to check and see if the values are present first